components/automate-ui/src/app/modules/infra-proxy/create-org-modal/create-org-modal.component.html

```diff
@@ -2,72 +2,81 @@
-          </label>
-          <chef-error *ngIf="createForm.get('id').hasError('maxlength') && createForm.get('id').dirty">
-            ID must be 64 characters or less.
-          </chef-error>
-          <chef-error *ngIf="createForm.get('id').hasError('required') && createForm.get('id').dirty">
-            ID is required.
-          </chef-error>
-          <chef-error *ngIf="createForm.get('id').hasError('pattern') && createForm.get('id').dirty">
-            Only lowercase letters, numbers, hyphens, and underscores are allowed.
-          </chef-error>
-          <chef-error *ngIf="conflictError">
-            {{ objectNoun | titlecase }} ID "{{createForm.get('id').value}}" already exists.
-          </chef-error>
-        </chef-form-field>
-        <span class="detail light">{{ objectNoun | titlecase }} IDs are unique, permanent, and cannot be changed later.</span>
-      </div>
-      <div *ngIf="!modifyID" class="id-margin">
-        <div id="id-fields">
-          <span class="key-label">ID:&nbsp;</span>
-          <span data-cy="id-label">{{ this.createForm?.value.id }}</span>
+      <div class="scrollable-form-div">
+        <div class="input-margin">
+          <chef-form-field>
+            <label>
+              <span class="label">Name <span aria-hidden="true">*</span></span>
+              <input chefInput name="name" formControlName="name" type="text" (keyup)="handleNameInput($event)" data-cy="org-name" autocomplete="off">
+            </label>
+            <chef-error
+              *ngIf="(createForm.get('name').hasError('required') || createForm.get('name').hasError('pattern')) && createForm.get('name').dirty">
+              Display Name is required.
+            </chef-error>
+          </chef-form-field>
+          <span class="detail light">Don't worry, organization names can be changed later.</span>
+        </div>
+        <div *ngIf="modifyID" class="id-margin">
+          <chef-form-field>
+            <label>
+              <span class="label">ID <span aria-hidden="true">*</span></span>
+              <input chefInput formControlName="id" type="text" (keyup)="handleInput($event)" id="id-input" data-cy="create-id" autocomplete="off"/>
+            </label>
+            <chef-error *ngIf="createForm.get('id').hasError('maxlength') && createForm.get('id').dirty">
+              ID must be 64 characters or less.
+            </chef-error>
+            <chef-error *ngIf="createForm.get('id').hasError('required') && createForm.get('id').dirty">
+              ID is required.
+            </chef-error>
+            <chef-error *ngIf="createForm.get('id').hasError('pattern') && createForm.get('id').dirty">
+              Only lowercase letters, numbers, hyphens, and underscores are allowed.
+            </chef-error>
+            <chef-error *ngIf="conflictError">
+              {{ objectNoun | titlecase }} ID "{{createForm.get('id').value}}" already exists.
```

Comment:
Copy/paste issue. You use this in a couple places but it is never defined.
```suggestion
              Organization ID "{{createForm.get('id').value}}" already exists.
```